openlibrary/core/vendors.py

```diff
@@ -2,6 +2,7 @@
 import web
 import urllib2
 import simplejson
```

Comment:
I'm suspicious of the `simplejson` reference. Does the API really return data XML encoded and errors JSON encoded?

If the data is actually returned as XML, why are we "parsing" it with regexs instead of an XML parser?

Reply:
@mekarpeles I've been finding `urllib2` references cause problems and can be replaced completely by `requests` (as it looks like you have done below.

The `simplejson` import is generally required with `urllib2` to parse the response, but that is not needed with `requests` as it has a built in `json()` -- I think replacing `urllib2` + `simplejson` with `requests` everywhere in the code is something we should aim for, it helps with Python3 and just makes network requests so much easier. Not sure it warrants a single push effort, but I think making it known that it's a common goal may help refactoring + moving forward with adding new features.

The two imports on this file and refactor should be removed.